### DISCUSSION TITLE
sir i am about to start prometheus practice but before that i have executed vpc and when i go to the vpc website it is showing earror but at the time of execution i have not got any earror

### DISCUSSION THREAD

**Discussion by vyshuvenu** [learndevopsonline/learndevopsonline#1495] [Q&A]
![Screenshot (49)](https://github.com/learndevopsonline/learndevopsonline/assets/139372307/94904e16-3683-4a72-b6be-0f419dbf3388)

/

**Comment by sreedharm07** [comment]
Hi @vyshuvenu ping your workstation ip

**Reply by vyshuvenu** [reply]
3.80.48.137

**Reply by sreedharm07** [reply]
Hi @vyshuvenu
there is no database things are running, you have disabled all database things in code,

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/94cb247f-a795-4535-bcee-26980e26504b)
